boards/arm/cxd56xx/spresense/scripts/Make.defs

```diff
@@ -35,6 +35,7 @@
 
 include ${TOPDIR}/.config
 include ${TOPDIR}/tools/Config.mk
+include ${TOPDIR}/boards/arm/cxd5xx/scripts/cxd56xx_Config.mk
```

Comment:
typo 
cxd56xx

Reply:
Thanks!  I have fixed that.
If we can get this verfied and merged, then we can deal with the similar issues for ESP32 next.